.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.4.4-dev0
+current_version = 0.4.5-dev1
```

Comment:
Wait. Why is 0.4.5-dev1? Did we release 0.4.4?

Reply:
yes